pkg/cmd/util/helper_test.go

```diff
@@ -162,29 +168,30 @@ max-size = 200
 max-days = 1
 max-backups = 1
 `, dataDir)
-	err := os.WriteFile(configPath, []byte(configContent), 0o644)
-	c.Assert(err, check.IsNil)
+	err = os.WriteFile(configPath, []byte(configContent), 0o644)
+	require.Nil(t, err)
 
 	conf := config.GetDefaultServerConfig()
 	err = StrictDecodeFile(configPath, "test", conf)
-	c.Assert(err, check.ErrorMatches, ".*contained unknown configuration options.*")
+	require.Contains(t, err.Error(), "contained unknown configuration options")
 }
 
-func (s *utilsSuite) TestAndWriteExampleReplicaTOML(c *check.C) {
-	defer testleak.AfterTest(c)()
+func TestAndWriteExampleReplicaTOML(t *testing.T) {
 	cfg := config.GetDefaultReplicaConfig()
 	err := StrictDecodeFile("changefeed.toml", "cdc", &cfg)
-	c.Assert(err, check.IsNil)
+	require.Nil(t, err)
 
-	c.Assert(cfg.CaseSensitive, check.IsTrue)
-	c.Assert(cfg.Filter, check.DeepEquals, &config.FilterConfig{
+	require.True(t, cfg.CaseSensitive)
+	require.Equal(t, &config.FilterConfig{
 		IgnoreTxnStartTs: []uint64{1, 2},
 		Rules:            []string{"*.*", "!test.*"},
-	})
-	c.Assert(cfg.Mounter, check.DeepEquals, &config.MounterConfig{
+	}, cfg.Filter)
+	require.Equal(t, cfg.Mounter, &config.MounterConfig{
```

Comment:
Is this order reversed? I remember that the expected value of this test framework comes first.

Reply:
`cfg.Mounter` is an actual value, it should be the third parameter in `require.Equal`. I will fix it.

Reply:
Are there other places you need to modify?

Reply:
> Are there other places you need to modify?

I have checked the other places. Only in this line the order is reversed.